aeon/schema/social_02.py

```diff
@@ -48,6 +48,12 @@ def __init__(self, path):
         super().__init__(_reader.Pose(f"{path}_test-node1*"))
 
 
+class Pose03(Stream):
```

Comment:
made add a comment that this is necessary due to changing registers for the pose streams for social02 in particular? And that 03 corresponds to the fact that this is because this pattern is what we're going with for social03 and moving forward? Or call this class something else?

Reply:
Or just import `Pose` from social_03.py as `Pose03`? I think I like something like this actually

Reply:
This is primarily due to full-pose data for social0.2 has a different pattern than that of the non-fullpose SLEAP data.
I don't think you can do `from social_03 import Pose as Pose03`, the way the the Devices/Streams being instantiated is that it's using the class name, so aliasing wouldn't work.

Reply:
ah I see, then just some documentation is ok